schema.sql

```diff
@@ -15,3 +15,54 @@ create table books (
   author_id integer references authors (id)  
 );
 
+-- a blog with users and posts
+-- a user has posts
+
+create table author (
+  id serial primary key,
+  name text
+);
+
+create table posts (
+  id serial primary key,
+  title text,
+  word_count integer,
+  category text,
+  publish_date timestamp,
+  author_id integer references author (id)
+);
+
+-- QUIZ APP ============================
+-- A quiz app, with questions and answers
+--a questions has an answer
+
+create table answer (
```

Comment:
Consider making these table names plural, to capture the idea "this is a thing that holds many records"